plugins/window_size/windows/window_size_plugin.cpp

```diff
@@ -177,6 +178,15 @@ void WindowSizePlugin::HandleMethodCall(
     SetWindowPos(GetRootWindow(registrar_->GetView()), NULL, x, y, width,
                  height, SWP_NOACTIVATE | SWP_NOOWNERZORDER);
     result->Success();
+  } else if (method_call.method_name().compare(kSetWindowTitleMethod) == 0) {
+    if (!method_call.arguments() || !method_call.arguments()->IsString()) {
+      result->Error("Bad arguments", "Expected string");
+      return;
+    }
+    const auto &title = method_call.arguments()->StringValue();
+    std::wstring stemp = std::wstring(title.begin(), title.end());
+    SetWindowTextW(GetRootWindow(registrar_->GetView()), stemp.c_str());
```

Comment:
You shouldn't need the W here; the project should be set to unicode, so `SetWindowText` would call the right one.

Reply:
You are right, the W is not required. With that said, the unicode pass through isn't really happy on Windows. Both macOS and Linux display the Japanese characters I've added, but Windows is messing it up. Help?

Reply:
Sorry, I didn't look closely enough at this code. This line:
```
std::wstring stemp = std::wstring(title.begin(), title.end());
```
is taking UTF-8 bytes and trying to read them as if they were UTF-16. You need to actually convert between the two; the reverse of [this](https://github.com/google/flutter-desktop-embedding/blob/67ee9712b7867369b100997db9f7048c0d55ba69/plugins/flutter_plugins/path_provider_fde/windows/path_provider_plugin.cpp#L32).

Reply:
Implemented, and tested. Thanks for the trail of breadcrumbs...